eng/pipelines/templates/steps/fetch-namespaces.yml

```diff
@@ -0,0 +1,31 @@
+parameters: 
+  DotnetNupkgLocations: ''
+  TargetBranch: ''
+  DocRepoLocation: ''
+  TargetDocRepoName: azure-docs-sdk-dotnet
+  TargetDocRepoOwner: Azure
+  WorkingDirectory: $(System.DefaultWorkingDirectory)
+steps:
+  - ${{ if eq(length(parameters.DotnetNupkgLocations), 0) }}:
```

Comment:
Don't we need an else statement for this if block? Otherwise I'm not sure I understand what it is doing.

Reply:
I am testing whether we could merge this step together with push package.json to docs repo. I will ping you once the PR is stable.